### PR TITLE
op-guide: Add load data compatibility description

### PR DESCRIPTION
**Pull request by zimulala**

### CODE REVIEW

File: op-guide/mysql-compatibility.md [hunk 1]
```diff
@@ -40,3 +40,6 @@ TiDB 实现了 F1 的异步 Schema 变更算法，DDL 执行过程中不会阻
 
 ### 事务
 TiDB 使用乐观事务模型，在执行 Update、Insert、Delete 等语句时，只有在提交过程中才会检查写写冲突，而不是像 MySQL 一样使用行锁来避免写写冲突。所以业务端在执行 SQL 语句后，需要注意检查 commit 的返回值，即使执行时没有出错，commit的时候也可能会出错。
+
+### Load data
+TiDB 在执行 load data 时，默认每 20 万行记录作为一个事务进行持久化存储。
```
Comment (shenli) [inline op-guide/mysql-compatibility.md]: 可以写的再详细些

**Comment by shenli** [comment]
LGTM

**Comment by shenli** [comment]
@coocood PTAL

**Comment by coocood** [comment]
LGTM

**Pull request merged**